### PR TITLE
Feature/rest api route

### LINKED ISSUE #162: Unify organizations and apis routes
Follow the recommendation from #86 and change the routes for 
- organization catalog
- individual organization profile
- API catalog
- individual API profile
- REST api

Also, consider the impact of the change.

### PR DESCRIPTION
**Pull request by brylie**
Closes #162 

# Changes
- Change api route to `/api`
- Remove hardcoded values from swagger definition
  - Apika specific language
  - Broken link to terms of service
- Make swagger fields more generic

### CODE REVIEW

File: core/server/api.js [hunk 1]
```diff
@@ -19,15 +19,7 @@ ApiV1.swagger = {
...
-      title: 'Apika API',
-      description: 'Apika REST API',
-      termsOfService: 'https://apika.digipalvelutehdas.fi/terms/',
-      contact: {
-        name: 'Apika team',
-      },
-      license: {
-        name: 'MIT',
-      },
+      title: 'Admin API',
```
Comment (jykae) [inline core/server/api.js]: @brylie make sure swagger.json is still valid if you remove most of the meta. Note also that currently REST API is not admin only. GET endpoints can be used by any logged in user.

Comment (brylie) [inline core/server/api.js]: Is the swagger.json *not* valid here?

http://swagger.io/specification/#infoObject

Comment (jykae) [inline core/server/api.js]: @brylie ok, tried also with validator, http://bigstickcarpet.com/swagger-parser/www/index.html Looks like it is valid.

Though if we call it "Admin API", we should restrict all endpoints 'admin' role only.
Is that requested?

Comment (brylie) [inline core/server/api.js]: That isn't the suggestion. I was just choosing a generic name that didn't use 'Apinf' or 'Apika'.

**Pull request merged**